src/runtime/cli/config/configuration-qemu.toml.in

```diff
@@ -56,6 +56,13 @@ firmware = "@FIRMWAREPATH@"
 # For example, `machine_accelerators = "nosmm,nosmbus,nosata,nopit,static-prt,nofw"`
 machine_accelerators="@MACHINEACCELERATORS@"
 
+# Qemu sandbox feature
+# comma-separated list of sandbox features to control the syscall access.
+# For example, `sandbox_on= "on,obsolete=deny,elevateprivileges=deny,spawn=deny,resourcecontrol=deny"`
+# Please note, enable this feature may lead some performance cost, you may enable
+# /proc/sys/net/core/bpf_jit_enable to reduce the impact.
+sandbox_on="@DEFSANDBOXPARAM@"
```

Comment:
This feature should be disabled by default as you mentioned, so you would need to comment out the line.

```
# sandbox_on="@DEFSANDBOXPARAM@"
```